gemrb/core/GUI/GameControl.cpp

```diff
@@ -50,6 +50,10 @@
 
 namespace GemRB {
 
+// A deadzone when rotating the formation, to prevent erratic movement near
+// the initial click.
+const int rotate_deadzone = 5;
```

Comment:
now that we use c++11 we should favor `constexpr` in cases like this

```suggestion
constexpr int rotate_deadzone = 5;
```